deployments/rbac-server/values.yaml

```diff
@@ -8,22 +8,24 @@ apiKeyCache:
   syncInterval: 10s
   userManagerServerInternalAddr: llm-operator-user-manager-server-internal-grpc:8082
 
+roleScopesMap:
+  owner:
+  - api.model.read
```

Comment:
Should we have a separate role for inference? Currently we use `api.model`, but I'm wondering if it should be `api.chat.completions`. 

https://github.com/llm-operator/inference-manager/blob/main/server/internal/server/server.go#L75

Reply:
Or `model.request` if we follow the openai.
Here is the openai's scope spec:
```
{
    "name": "Models",
    "description": "List models this organization has access to",
    "permissions_to_scopes": {
      "read": [
        "model.read",
        "api.model.read"
      ]
    },
    "endpoints": [
      "/v1/models"
    ]
},
{
    "name": "Model capabilities",
    "description": "Create chat completions, audio, embeddings, and images",
    "permissions_to_scopes": {
      "write": [
        "model.request",
        "api.model.request"
      ]
    },
    "endpoints": [
      "/v1/audio",
      "/v1/chat/completions",
      "/v1/embeddings",
      "/v1/images",
      "/v1/moderations"
    ]
  },
```

Reply:
I see. Sounds good!